app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
     <% end %>
     <meta name="author" content="Public Lab contributors" />
     <link href="https://<%= request.host %>/feed.rss" rel="alternate" type="application/rss+xml" title="Public Lab research" />
+    <link href="/lib/noty/lib/noty.css" rel="stylesheet">
+<script src="/lib/noty/lib/noty.js" type="text/javascript"></script>
```

Comment:
And if this is on every page, we could get it compiled in, at `/app/assets/javascripts/application.js` and `/app/assets/stylesheets/application.css`